File: MyApplication/app/src/main/java/com/example/myapplication2/Examen_Final.java
package com.example.myapplication2;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class Examen_Final extends AppCompatActivity {

    Button Ejercicio_1, Ejercicio_2, Ejercicio_3;
//    TextView TextViewResultado;
//    EditText editTextA, editTextB;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_examen_final);

//        editTextA = findViewById(R.id.editTextA);
        Ejercicio_1 = findViewById(R.id.button_eje1);
        Ejercicio_2 = findViewById(R.id.button_eje2);
//        Ejercicio_3 = findViewById(R.id.ButtonOperaciones);
//        Ejercicio_4 = findViewById(R.id.ButtonOperaciones);


//        ButtonGraficos2DTarea.setOnClickListener(new View.OnClickListener() {
//            @Override
//            public void onClick(View v) {
//                setContentView(new Graficos2DTarea(Examen_Final.this));
//            }
//        });


        Ejercicio_1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(Examen_Final.this, eje_1_div.class);
                startActivity(intent);
            }
        });
//
//
        Ejercicio_2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(Examen_Final.this, WebService4Activity.class);
                startActivity(intent);
            }
        });
//
//
//        Ejercicio_3.setOnClickListener(new View.OnClickListener() {
//            @Override
//            public void onClick(View v) {
//                Intent intent = new Intent(Examen_Final.this, Ecuaciones.class);
//                startActivity(intent);
//            }
//        });
//
//
//        Ejercicio_4.setOnClickListener(new View.OnClickListener() {
//            @Override
//            public void onClick(View v) {
//                Intent intent = new Intent(Examen_Final.this, Ecuaciones.class);
//                startActivity(intent);
//            }
//        });


    }
}

File: kotlin_proyect/app/src/main/java/com/example/kotlin_proyect/Examen_final.kt
package com.example.kotlin_proyect

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

class Examen_final : AppCompatActivity() {
//    Lo de sqlite se encuentra en la siguientes carpetas:
//    Lugares.kt, BaseDatos.kt, DB_activity

//    Lo de webservices esta en:
//    Wscompras,wspersons, wsservicios, wsventas, webservicesApi, mainAcitvity

//    ResponseLugar.kt, webservicesApi, MainActivity
//
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_examen_final)

        val Button_Eje1 = findViewById<Button>(R.id.button_Eje1)
//        val Button_Eje2 = findViewById<Button>(R.id.button_Eje2)
//        val Button_Eje3 = findViewById<Button>(R.id.button_Eje3)
//        val Button_Eje4 = findViewById<Button>(R.id.button_Eje4)
//       val editB = findViewById<EditText>(R.id.editB)
//       val textA = findViewById<TextView>(R.id.textA)
//      usar como ejemplo practtica_1_course


        Button_Eje1.setOnClickListener(View.OnClickListener { setContentView(cubo(this@Examen_final)) })
//
//        ButtonTriangulo_color.setOnClickListener(View.OnClickListener { setContentView(triangulo_color(this@Examen_final)) })


//        Button_Eje1.setOnClickListener(){
//            val intent = Intent(this@Examen_final, cubo::class.java)
//            startActivity(intent)
//
//        }
//
//        Button_Eje2.setOnClickListener(){
//            val intent = Intent(this@Examen_final, DB_Activity::class.java)
//            startActivity(intent)
//
//        }
//
//        Button_Eje3.setOnClickListener(){
//            val intent = Intent(this@Examen_final, DB_Activity::class.java)
//            startActivity(intent)
//
//        }
//
//        Button_Eje4.setOnClickListener(){
//            val intent = Intent(this@Examen_final, DB_Activity::class.java)
//            startActivity(intent)
//
//        }
    }
}